docs/operation/operation.md

```diff
@@ -0,0 +1,169 @@
+
+**Navigation**
+
+- [Background](../index.md)
+- [Contribution](../contribution/contribution.md) 
+
+---
+
+# Operational Docs
+
+Welcome to the Operational Documentation for setting up and managing the Score Server in a development environment. This document provides detailed steps and configurations needed to get your development environment up and running with either a standalone setup or using Docker. This page also includes instructions for integrating Keycloak for authentication and authorization.
```

Comment:
Don't think we need welcome statements, but i'll leave that call to you.

Reply:
Agreed, bit cheesy, contrived and unnecessary